{"name": "test06", "status": "passed", "description": "assert that you get a exception when you type invalid email adress", "start": 1710320851232, "stop": 1710320854350, "uuid": "89acda9f-c485-49ee-b282-882c8fadb22c", "historyId": "b9b530c5cc25f8123ba9c34a7ef66853", "testCaseId": "b9b530c5cc25f8123ba9c34a7ef66853", "fullName": "test_cases.Test_cases#test_assert_exception_message_when_type_invalid_email", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_webdriver')"}, {"name": "suite", "value": "test_cases"}, {"name": "subSuite", "value": "Test_cases"}, {"name": "host", "value": "LAPTOP-C1F79DIL"}, {"name": "thread", "value": "6260-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases"}]}